core/templates/components/common-layout-directives/navigation-bars/top-navigation-bar.directive.ts

```diff
@@ -142,7 +138,22 @@ angular.module('oppia').directive('topNavigationBar', [
             return SidebarStatusService.isSidebarShown();
           };
           ctrl.toggleSidebar = function() {
-            SidebarStatusService.toggleSidebar();
+            /**
+             * Note to the developer migrating this directive:
+             * I (@ashutoshc8101) have defined the toggle function in an angular
```

Comment:
Don't put `I` or name here. We can always look at git blame to figure out who was the author. Just keep the explanation for why the toggle was introduced.

Reply:
done